hack/lib/serverless.bash

```diff
@@ -151,6 +153,32 @@ EOF
   logger.success 'Knative Eventing has been installed sucessfully.'
 }
 
+function deploy_knativekafka_cr {
+  logger.info 'Deploy Knative Kafka'
+
+  # Wait for the CRD to appear
+  timeout 900 "[[ \$(oc get crd | grep -c knativekafkas.operator.serverless.openshift.io) -eq 0 ]]" || return 6
+
+  # Install Knative Kafka
+  cat <<EOF | oc apply -f - || return $?
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: ${EVENTING_NAMESPACE}
+spec:
+  source:
+    enabled: true
+  channel:
+    enabled: true
+    bootstrapServers: foo.bar.example.com:1234
```

Comment:
Let's not enable this - especially w/ the bootstrap server this by default

Reply:
sounds good.

Reply:
no channel installation to be done here in this step.

Reply:
I changed this to url of the cluster that Strimzi creates.

Let's leave for now. Once we're in real condition to merge this PR, we can discuss options.

Reply:
Because:
- this script is only used for devl purposes
- disabling the channel would mean we need to patch it later when doing e2e tests (checking deployment shapes etc) and upstream e2e tests (actually running eventing-contrib e2e tests)